bee-ternary/src/lib.rs

```diff
@@ -59,25 +59,25 @@ where
         unsafe { &*(T::empty() as *const _ as *const Self) }
     }
 
-    pub unsafe fn from_raw_unchecked(raw: &[i8]) -> &Self {
-        &*(T::from_raw_unchecked(raw) as *const _ as *const _)
+    pub unsafe fn from_raw_unchecked(raw: &[i8], num_trits: usize) -> &Self {
+        &*(T::from_raw_unchecked(raw, num_trits) as *const _ as *const _)
```

Comment:
Might be worth adding a `debug_assert` here to make sure `num_trits` isn't too long for the `&[i8]`.

Reply:
sure